third_party/sycl/crosstool/CROSSTOOL.tpl

```diff
@@ -177,19 +163,20 @@ toolchain {
 
   compiler_flag: "-target"
   compiler_flag: "%{ARM_TARGET}%"
-  compiler_flag: "-isystem"
-  compiler_flag: "%{PYTHON_INCLUDE_PATH}%"
-  compiler_flag: "-DEIGEN_HAS_C99_MATH=1"
   compiler_flag: "--gcc-toolchain=%{ARM_COMPILER_PATH}%"
   compiler_flag: "--sysroot=%{ARM_COMPILER_PATH}%/%{ARM_TARGET}%/libc"
+
   cxx_flag: "-std=c++11"
   cxx_flag: "-isystem"
+  cxx_flag: "%{PYTHON_INCLUDE_PATH}%"
+  cxx_flag: "-isystem"
   cxx_flag: "%{COMPUTECPP_ROOT_DIR}%/include"
   cxx_flag: "-isystem"
   cxx_flag: "%{OPENCL_INCLUDE_DIR}%"
   cxx_flag: "-fsycl-ih-last"
   cxx_flag: "-sycl-driver"
-  cxx_flag: "-no-serial-memop"
+  # Serialisation of memory operations is required on some platforms
+  #cxx_flag: "-no-serial-memop"
```

Comment:
As above I think we should just remove this and let folks add it when they need it